draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -184,6 +183,7 @@ Consistency proofs provide a mechanism for a verifier to validate the consistenc
 
 The integer identifier for this Proof Type is TBD_3.
 The string identifier for this Proof Type is "consistency".
+The value of this Proof Type is bstr or list of bstr.
```

Comment:
Wouldn't it be better/simpler for this to ALWAYS be a list and sometimes only have 1 element?